applications/slonigiraf.md

```diff
@@ -0,0 +1,186 @@
+# W3F Grant Proposal
+
+- **Project Name:** Slonigiraf
+- **Team Name:** Slonigiraf
+- **Payment Address:** bc1qyuk4rdeqvc2rmetfhr3key4clty5sglaw4n5wx (BTC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+- Slonigiraf is an p2p education system.
+- We want to bring to Web3 infrastructure the renowned Slonigiraf learning game, which has been designed to deliver more
+  interactive and productive lessons through peer-to-peer learning and quality control through cross-exams. This game in
+  a nutshell consists of the following parts:\
+  Sara (student) pays Tanya (teacher) to teach skill_1 and issue a certificate that Sarah has this skill. Such a
+  certificate not only shows that Sarah has skill_1, but also gives Sarah the ability to allow other people fine Tania a
+  fixed amount of money if Sarah lacks the skill. The penalty could be applied only once.\
+  Sarah then goes to another teacher Teri and pays him to teach skill_2 and get a certificate. Sarah provides Teri the
+  opportunity to revise skill_1 and penalize Tania.\
+  Sarah then goes to Ed (the employer) to get a job that requires skill_1 and skill_2. Sarah presents Ed with
+  certificates and give Ed a chance to punish Tanya and Terry. If Ed believes Sarah has the skills, he will hire Sarah
+  and also pay Sarah for the certificates use. Otherwise, Ed may penalize teachers for incorrect certificates.\
+  SLON works as a reputation token. You could be surprised that we provide a possibility to buy reputation but this design
+  really works - since you pay for your reputation, you value it.\
+  The relation between SLONs and guarantees is following: teacher can issue 5 guarantees having 100 SLONs on his account, 
+  but each guarantee, if penalized, costs teacher 100 SLON.\
+  All parties: Sarah, Tanya, Teri and Ed are economically motivated to behave honestly.\
+  Ed is economic incentivized penalize a teacher in case if Sarah lack the skill because he gets SLON from such teacher.\
+  But also Ed can't behave dishonestly. If he starts to penalize teachers in case when skill is present - he will lose
```

Comment:
The reasoning here is not sound, and your application also does not describe how Ed attains an identity on the chain that entitles him to slash teachers. More generally, I struggle to see anything in this application that actually enforces the desired behaviours, not to mention prevention of bribery.

Reply:
Let's consider different ways of misbehavior of parties.
We should note here that Sara and Tanya, Sara and Teri are persons who trust each other, and Ed is a public company that Sara trusts. This "trust" is an essential composite of the system. Teacher will not issue a certificate for a student whom he doesn't trust. A student will not show a certificate to a company that is known for misbehavior.
A). Sarah gets a certificate from Tanya but then intentionally tells Teri that she lack such skill. Teri penalize Tanya and Tanya loses some amount of SLON. As Tanya knows exactly that Sarah is good at the skill then Tanya decides not to work any more with Sarah and not to issue other certificates for Sarah. Sarah loses faith in herself in Tanya's mind. And this "faith" cost a lot. By the way Sarah has no motif to act in such way.
B). The same as above but Sarah allows Ed to penalize Tanya. Sarah will not get a work if she do so. It's not profitable for Sarah.
C). Ed knows that Sarah has a skill but penalize a teacher. He will lose all new workers - nobody wants to work with unfair hirer D). Teri knows that Sarah has a skill but penalize Tanya. Sarah knows that Teri cheated and will not work with him in future, Teri will lose a profit from issuing future certificates for Sarah.
E). Sarah bribes a teacher to issue a certificate. Ed will examine her and penalize the teacher. The pennalization sum is larger that Sarah's bribe. Thus the teacher can't afford to get a bribe from a student.